### PR TITLE
Update to use go1.12.12

### PR DESCRIPTION
**Pull request by cblecker**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
ref #84065

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
Update to use go1.12.12
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by tao12345666333** [comment]
Should we update to 1.12.12 directly?

<p>
go1.12.11 (released 2019/10/17) includes security fixes to the
<code>crypto/dsa</code> package.
See the <a href="https://github.com/golang/go/issues?q=milestone%3AGo1.12.11">Go
1.12.11 milestone</a> on our issue tracker for details.
</p>

<p>
go1.12.12 (released 2019/10/17) includes fixes to the go command,
runtime, <code>syscall</code> and <code>net</code> packages.
See the <a href="https://github.com/golang/go/issues?q=milestone%3AGo1.12.12">Go
1.12.12 milestone</a> on our issue tracker for details.
</p>

**Comment by fejta** [comment]
```
ERROR: error loading package '': Encountered error while reading extension file 'go/deps.bzl': no such package '@io_bazel_rules_go//go': com.google.devtools.build.lib.syntax.EvalException: Definition of repository io_bazel_rules_go: Syntactically invalid SHA256 checksum: '5c02dd4436332f8efa982ce2102daf7c041ae95c2bc7f2d26aceed20dc47dxxx' at /home/prow/go/src/github.com/kubernetes/kubernetes/WORKSPACE:45:1
```

**Comment by cblecker** [comment]
@javier-b-perez @BenTheElder `golang:1.12.12` is live. This is ready for cross image build.

**Comment by javier-b-perez** [comment]
working on the images now
/lgtm

File: build/build-image/cross/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v1.12.10-1
+v1.12.12-1
```
Comment (javier-b-perez) [inline build/build-image/cross/VERSION]: should this be 12.12-0 ?

Comment (javier-b-perez) [reply build/build-image/cross/VERSION]: nm, sees we start in 1 always

**Comment by BenTheElder** [comment]
xref: https://github.com/kubernetes-sigs/kind/pull/963 seems the bazel `--platforms` flag is broken by this, but only kind was using that, and we don't actually need it since cross building kubelet doesn't work anyhow (cgo + rules_go ...), that PR should be in soon and the test is currently not blocking.

**Comment by BenTheElder** [comment]
@cblecker needs another moment, k/k presubmits consume pre-built binaries to save time https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kind-unit/1185289466755944453

**Comment by BenTheElder** [comment]
looks like e2e-gce is not coming up on this PR though?? https://prow.k8s.io/pr-history/?org=kubernetes&repo=kubernetes&pr=84064

**Comment by BenTheElder** [comment]
> panic: version string "{gitVersion}" doesn't match expected regular expression: "^v(\d+\.\d+\.\d+)"
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/84064/pull-kubernetes-e2e-gce/1185269718089469953/artifacts/e2e-a642edb3d2-674b9-master/kubelet.log

:confused:

**Comment by BenTheElder** [comment]
xref https://github.com/bazelbuild/rules_go/issues/2224

**Comment by BenTheElder** [comment]
/lgtm
:crossed_fingers:  hopefully the stamp works.

**Comment by BenTheElder** [comment]
kind and GCE tests up and running now 👍

**Pull request merged**